rules/windows/defense_evasion_masquerading_suspicious_werfault_childproc.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/08/24"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+A suspicious WerFault child process was detected. This behavior may indicate an attempt to run unnoticed. Verify process
+details such as commandline, network connections, file writes and parent process details as well.
+"""
+false_positives = ["Custom Windows Error Reporting Debugger"]
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious WerFault Child Process"
+references = [
+    "https://www.hexacorn.com/blog/2019/09/19/silentprocessexit-quick-look-under-the-hood/",
+    "https://github.com/sbousseaden/EVTX-ATTACK-SAMPLES/blob/master/Persistence/persistence_SilentProcessExit_ImageHijack_sysmon_13_1.evtx",
+]
+risk_score = 47
+rule_id = "ac5012b8-8da8-440b-aaaf-aedafdea2dff"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category :"process" and not event.type:"end" and
+ process.parent.name:"WerFault.exe"  and
+ not process.pe.original_file_name:(cofire.exe or psr.exe or VsJITDebugger.exe or TTTracer.exe or RUNDLL32.EXE)
```

Comment:
nit:

```suggestion
event.category:process and not event.type:end and
 process.parent.name:WerFault.exe  and
 not process.pe.original_file_name:(cofire.exe or psr.exe or VsJITDebugger.exe or TTTracer.exe or RUNDLL32.EXE)
```

Reply:
As of 7.8.0, winlogbeat did not use the `process.pe.original_file_name` field. Will need to verify if it is being used as of 7.9.0 or else use the field `winlog.event_data.OriginalFileName` for winlog compatibility